sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/protocol/SdkVersion.kt

```diff
@@ -0,0 +1,20 @@
+package io.sentry.kotlin.multiplatform.protocol
+
+actual data class SdkVersion actual constructor(
+    actual val name: String,
+    actual val version: String,
+    actual val packages: MutableList<Package>
```

Comment:
```suggestion
    actual val packages: List<Package> = mutableListOf()
```

Can we do this instead? or even make it nullable